.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Set up Python
+        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
+        with:
+          python-version: 3.12
+
+      - name: Build package using ros_release_python
+        run: |
+          # Create a temporary directory for ros_release_python
+          TEMP_DIR=$(mktemp -d)
+          cd $TEMP_DIR
+
+          # Clone the ros_release_python repository
+          git clone https://github.com/ros-infrastructure/ros_release_python.git
+
+          # Build the docker image
+          cd ros_release_python
+          docker build -t rrp .
```

Comment:
I think that future work could involve building the rrp container and uploading it to GitHub Packages to be used here rather than building it inline (this becomes extra valuable when expanding this to other projects)